### DISCUSSION TITLE
测试版隐藏应用列表

### DISCUSSION THREAD

**Discussion by Laomaisb** [community/community#33916] [Other Feature Feedback, Questions, & Ideas]
In Android 11+, turning on the hidden application list setting (forcibly binding and mounting data memory) will invalidate the virtual positioning of the soft-modified software. I hope the author can fix it after seeing this message. Thank you very much!

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file,](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project) if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
